<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="21852" started-at="2020-07-13T11:55:27 IST" finished-at="2020-07-13T11:55:49 IST">
    <groups>
    </groups>
    <test name="Test" duration-ms="21852" started-at="2020-07-13T11:55:27 IST" finished-at="2020-07-13T11:55:49 IST">
      <class name="com.CURA.Testcase.Login_CURA">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.CURA.Testcase.Login_CURA@25d250c6]" name="setUp" is-config="true" duration-ms="9830" started-at="2020-07-13T11:55:27 IST" finished-at="2020-07-13T11:55:37 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="PASS" signature="loginApp()[pri:0, instance:com.CURA.Testcase.Login_CURA@25d250c6]" name="loginApp" duration-ms="8298" started-at="2020-07-13T11:55:37 IST" finished-at="2020-07-13T11:55:46 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginApp -->
        <test-method status="PASS" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.CURA.Testcase.Login_CURA@25d250c6]" name="tearDownMethod" is-config="true" duration-ms="15" started-at="2020-07-13T11:55:46 IST" finished-at="2020-07-13T11:55:46 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginApp status=SUCCESS method=Login_CURA.loginApp()[pri:0, instance:com.CURA.Testcase.Login_CURA@25d250c6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method status="FAIL" signature="loginApp_test1()[pri:0, instance:com.CURA.Testcase.Login_CURA@25d250c6]" name="loginApp_test1" duration-ms="2496" started-at="2020-07-13T11:55:46 IST" finished-at="2020-07-13T11:55:48 IST">
          <exception class="java.lang.ArrayIndexOutOfBoundsException">
            <message>
              <![CDATA[10]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.ArrayIndexOutOfBoundsException: 10
at jxl.read.biff.SheetImpl.getCell(SheetImpl.java:356)
at Utilities.ExcelDataProvider.readExcel(ExcelDataProvider.groovy:25)
at org.codehaus.groovy.vmplugin.v7.IndyInterface.selectMethod(IndyInterface.java:234)
at com.CURA.Testcase.Login_CURA.loginApp_test1(Login_CURA.groovy:87)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:821)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1257)
at org.testng.TestRunner.privateRun(TestRunner.java:767)
at org.testng.TestRunner.run(TestRunner.java:588)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1214)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1136)
at org.testng.TestNG.runSuites(TestNG.java:1066)
at org.testng.TestNG.run(TestNG.java:1034)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.ArrayIndexOutOfBoundsException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- loginApp_test1 -->
        <test-method status="PASS" signature="tearDownMethod(org.testng.ITestResult)[pri:0, instance:com.CURA.Testcase.Login_CURA@25d250c6]" name="tearDownMethod" is-config="true" duration-ms="482" started-at="2020-07-13T11:55:48 IST" finished-at="2020-07-13T11:55:49 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=loginApp_test1 status=FAILURE method=Login_CURA.loginApp_test1()[pri:0, instance:com.CURA.Testcase.Login_CURA@25d250c6] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDownMethod -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.CURA.Testcase.Login_CURA@25d250c6]" name="tearDown" is-config="true" duration-ms="689" started-at="2020-07-13T11:55:49 IST" finished-at="2020-07-13T11:55:49 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.CURA.Testcase.Login_CURA -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
